indra/newview/llimview.cpp

```diff
@@ -3160,7 +3160,7 @@ void LLIMMgr::addMessage(
 			}
 
             // Fetch group chat history, enabled by default.
-            if (gSavedPerAccountSettings.getBOOL("FetchGroupChatHistory"))
+            if (gSavedPerAccountSettings.getBOOL("FetchGroupChatHistory") && gAgent.getRegion())
```

Comment:
There is a getRegionCapability that does this check inside, also there should be a !chat_url.empty() check in case region has no caps or no this specific cap.

Reply:
Thanks.